tailwind.config.cjs

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  content: ['./src/**/*.{js, jsx}'],
```

Comment:
You can add a purge:[ ] option so as to discard unused css

Reply:
@Jkadhuwa 
Tailwind automatically purge CSS in production. Added two new scripts to test the difference and ensured that build is run with mode set to production.
[Reference](https://v2.tailwindcss.com/docs/optimizing-for-production#:~:text=now%20whenever%20you%20compile%20your%20css%20with)

> And I changed the configuration to extend on the tailwind configuration for both Font family and colors